gui.js

```diff
@@ -4472,22 +4511,6 @@ IDE_Morph.prototype.getURL = function (url) {
     }
 };
 
-IDE_Morph.prototype.getURLsbeOrRelative = function (url) {
```

Comment:
Was the use of this function when loading Snap! In the browser from an HTML with a `file://` url?
In that case, the `baseURL` solution still doesn't work, but it's using the same URL. We would actually need to keep a hardcoded reference to s.b.e/snapsource

Do we want to support loading from a `file://` URL? There are plenty of other exceptions that simply can't be handled. I feel like a desktop editor is a better way to support this... Besides if you have a local snap.html download, you have all the files already available via the import function, or drag and drop.